packages/app/src/metadata-viewer/FiltersInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Filter } from '@h5web/shared';
+
+interface Props {
+  filters: Filter[];
+}
+
+function FiltersInfo(props: Props) {
+  const { filters } = props;
+
+  return (
+    <>
+      {filters.map((filter, i) => {
+        const { name, id } = filter;
+
+        return (
+          <tr key={id}>
+            <th scope="row">{`Filter ${i}`}</th>
+            <td>{`${name} (HDF5 id: ${id})`}</td>
```

Comment:
Not sure showing the index of the filter is worth it. Isn't the order sufficient? You could remove the `th` and have `colspan="2"` on the `td`, or you could move the filters to the dataset's core metadata table, on a single row (maybe with an `ol` inside the `td`).

Reply:
Or you could put the HDF5 ID in the `th`, maybe people will know what it's about without further clue?